### PR TITLE
Fix bug on hide logout menu

### PR DESCRIPTION
**Pull request by StephenAraka**
### what does this PR do?
This fixes a bug on the logout dropdown menu.

The menu could not close from clicking the down arrow.
Peek 2020-06-18 13-47

### Corresponding Trello ticket
https://trello.com/c/gyBWj4y6

### CODE REVIEW

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -81,6 +87,13 @@ div.HeaderLinksWrap.LoggedIn .BelowHeader {
   font-weight: bold;
   color: #fff;
   cursor: default;
+  -webkit-touch-callout: none; /* iOS Safari */
```
Comment (mrwagaba) [inline src/components/Header/Header.css:90]: @StephenAraka please give us some education on these rules

Comment (StephenAraka) [reply src/components/Header/Header.css:90]: **My understanding:** Different browsers and their versions are built and run on different browser engines. e.g WebKit for Safari (and Chrome - I'm not really sure), Gecko (Mozilla), EdgeHTML (Edge), KHTML (was widely used by a lot of browsers - Google Chrome, Safari, Opera, Vivaldi, Microsoft Edge until about 2016)...

When building a webapp, you never know what browser your users will use so, at times you need to write css properties that are meant to target specific browser engines.

For example the generic `user-select: none` may not work on Safari, IE etc...  so `-webkit-user-select: none` targets Safari, `-ms-user-select: none;` targets IE/Edge etc.

Comment (mrwagaba) [reply src/components/Header/Header.css:90]: we shall have a conversation on this

**Comment by mrwagaba** [comment]
had to resolve on here

**Pull request merged**